src/ontodia/widgets/classTree.tsx

```diff
@@ -127,7 +127,7 @@ function mapClass(
     const children = derived.map(child => mapClass(child, view, iconMap));
     const {icon} = view.getTypeStyle([id]);
 
-    const text = view.getLocalizedText(label).text + (Number.isNaN(count) ? '' : ` (${count})`);
+    const text = view.getLocalizedText(label).text + (isNaN(count) ? '' : ` (${count})`);
```

Comment:
Don't replace `Number.isNaN` with `isNaN` (they differ in semantics for non-number values); create a polyfill for `Number.isNaN` instead.